content/en/releases/download.md

```diff
@@ -68,7 +68,9 @@ container image name, for example
 [`k8s.gcr.io/kube-apiserver-arm64:{{< param "fullversion" >}}`][0-arm64]. All
 those derivations are signed in the same way as the multi-architecture manifest lists.
 
-<!-- TODO: describe how to use the signed images -->
+### Verifying Container Image Signatures
```

Comment:
tip: this should be a signpost into a page within https://kubernetes.io/docs/tasks/